American-Elections-and-Crypto.md

```diff
@@ -0,0 +1,99 @@
+
+“I'm a big fan of our currency and I don't want to have other currencies coming out and hurting or demeaning the dollar in any way. If you look at a monetary system based on the dollar, if you start losing credibility, all of a sudden you're going to lose that strong monetary system, we have to be very careful about that.” (https://finance.yahoo.com/news/trump-on-crypto-im-a-big-fan-of-the-dollar-warns-about-loss-of-status-in-global-finance-112730063.html#:~:text=%E2%80%9CI'm%20a%20big%20fan,the%20greenback%20is%20a%20fulcrum.)
+
+Trump also criticized Bitcoin during a Fox News interview in the same year. Showing once more a vision of protectionism when regarding the American dollar:
+
+"The currency of this world should be the dollar. And I don't think we should have all of the Bitcoins of the world out there. I think they should regulate them very, very high." (https://www.foxbusiness.com/markets/trump-bitcoin-a-scam-us-dollar-should-reign)
+
+The notion that cryptocurrencies should be heavily regulated is not favored by the majority of crypto investors. Given these remarks, it's challenging to predict how Trump would act in relation to crypto. While his involvement in launching digital assets like NFTs and holding crypto may suggest a favorable stance, it's impossible to make a definitive prediction.
+
+## Crypto in the last elections
+
+During the 2016 election, which marked the election of Trump, the cryptocurrency market had yet to reach its peak hype. While the crypto community was sizable, it hadn't attracted as many participants as it would in later years. Personally, I began investing during the significant boom of 2017. Bitcoin experienced notable appreciation during this time. For instance, on October 1st, one BTC was valued at $613, and by the end of November, it had risen to $750—a significant pump, though not comparable to what occurred during the 2020 elections.
+
+In contrast, during the 2020 elections, which saw Biden's election, Bitcoin experienced a remarkable surge. On October 1st, one BTC was priced at $10,500, soaring to nearly $19,000 by the end of November—an appreciation of almost 90%.
+
+These data suggest that while elections don't necessarily destabilize the crypto market, they also don't guarantee a significant pump. The crypto industry has evolved significantly, and market dynamics are influenced by various factors beyond political events.
+
+## Will anything really change?
+
+Both the Biden and Trump administrations, while having divergent views on cryptocurrency, share a common objective: to ensure that traditional financial institutions retain control and influence over the financial landscape, even as digital currencies gain traction. Despite potential differences in regulatory approaches, both administrations are likely to prioritize measures that maintain the supremacy of the American dollar and protect the interests of established financial entities.
```

Comment:
"Despite potential differences in regulatory approaches"

Getting to this point in the article, I'm still none the wiser as to what regulation policies either are likely to implement or reject. The status quo doesn't make for a very interesting read.

I think we need to find official documents that lays out their overall plans and see if we can ascertain if there is anything in them that relates to crypto. Without this information this article is pretty light of detail.

Reply:
Thanks a lot for the review. I'll try to find more official documents that can prove what is written. I will go through all of your comments